app/views/events/index.html.erb

```diff
@@ -15,10 +16,10 @@
         <th scope="col">Actions</th>
       </tr>
     </thead>
-
     <tbody>
       <% @events.each do |event| %>
         <tr>
+          <td><%= image_tag event.image.thumb if event.image? %></td>
```

Comment:
The default to this shouldn't be to have nothing (it looks weird on the view). There should be something that shows up if they don't upload a photo. This is especially true in the view for show.

Reply:
Should events not allowed to be saved without a photo? Or should I just have a default image?